# Software

- **B-Lightning**: Using bait genes for marker gene hunting in single-cell data with complex heterogeneity.
  - [paper](https://academic.oup.com/nar/article/52/9/e46/6668556)
  - [Code](https://github.com/yirenshao/B-Lightning)

- **xMINT**: A Multimodal Integration Transformer for Xenium Gene Imputation.
  - [paper](https://openreview.net/forum?id=hnYLq2lwOv) 
  - [Code](https://github.com/jichunxie/xMINT_manu_support)


- **SifiNet**: A robust and accurate method to identify feature gene sets and annotate cells.
  - [paper](https://academic.oup.com/nar/article/52/9/e46/6668556) 
  - [Code](https://github.com/jichunxie/sifinet)

- **DYNATE**: Localizing Rare-Variant Association Regions via Multiple Testing Embedded in an Aggregation Tree.
  -  [paper](https://onlinelibrary.wiley.com/doi/10.1002/gepi.22542?af=R)
  -  [Code](https://github.com/jichunxie/DYNATE)

- **DART**: Distance Assisted Recursive Testing
  - [paper](https://efaidnbmnnnibpcajpcglclefindmkaj/https://www.jmlr.org/papers/volume24/22-1131/22-1131.pdf) 
  - [Code](https://github.com/jichunxie/DART)

- **TEAM**: A multiple testing algorithm on the aggregation tree for flow cytometry analysis.
  - [paper](https://projecteuclid.org/journals/annals-of-applied-statistics/volume-17/issue-1/TEAM--A-multiple-testing-algorithm-on-the-aggregation-tree/10.1214/22-AOAS1645.short) 
  - [Code](https://github.com/jichunxie/TEAM)

- **CDI**: Clustering Deviation Index (CDI): A robust and accurate internal measure for evaluating scRNA-seq data clustering.
  - [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02825-5) 
  - [Code](https://github.com/jichunxie/CDI)

- **SQUAC**: An efficient and robust large-scale multiple testing method to infer high-dimensional gene co-expression networks based on sample-quantile contingency tables.
  - [paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6497089/)
  - [Code](https://github.com/jichunxie/squac)

- **bcSeq**: This package implements a highly efficient data structure and algorithm for performing alignment of short reads from CRISPR or shRNA screens to reference barcode library. Sequencing error are considered and matching qualities are evaluated based on Phred scores. A Bayes' classifier is employed to predict the originating barcode of a read.
  - [paper](https://academic.oup.com/bioinformatics/article/34/20/3581/5001385) 
  - [Code](https://github.com/jichunxie/bcSeq)

- **capme**: An efficient algorithm to estimate covariate-adjusted precision matrices.
  - [paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5351557/)
  - [Code](https://github.com/jichunxie/capme)

- **qlspack**: Fitting mixed effect model based on quasi-least-square.
  - [paper](https://core.ac.uk/download/pdf/61321077.pdf)
  - [Code](https://github.com/jichunxie/qlspack)