<!DOCTYPE html>
<html lang="en-US">
  <head>
    <meta charset="UTF-8">

<!-- Begin Jekyll SEO tag v2.8.0 -->
<title>Software | Welcome to Xie Lab!</title>
<meta name="generator" content="Jekyll v3.9.3" />
<meta property="og:title" content="Software" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Jichun Xie, Associate Professor, Duke University" />
<meta property="og:description" content="Jichun Xie, Associate Professor, Duke University" />
<link rel="canonical" href="http://localhost:4000/software.html" />
<meta property="og:url" content="http://localhost:4000/software.html" />
<meta property="og:site_name" content="Welcome to Xie Lab!" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Software" />
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"WebPage","description":"Jichun Xie, Associate Professor, Duke University","headline":"Software","url":"http://localhost:4000/software.html"}</script>
<!-- End Jekyll SEO tag -->

    <link rel="preconnect" href="https://fonts.gstatic.com">
    <link rel="preload" href="https://fonts.googleapis.com/css?family=Open+Sans:400,700&display=swap" as="style" type="text/css" crossorigin>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="theme-color" content="#157878">
    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
    <link rel="stylesheet" href="/assets/css/style.css?v=ce1a2475d73e9ff1ec431a77af17b6de1bc4d7b6">
    <!-- start custom head snippets, customize with your own _includes/head-custom.html file -->

<!-- Setup Google Analytics -->



<!-- You can set your favicon here -->
<!-- link rel="shortcut icon" type="image/x-icon" href="/favicon.ico" -->

<!-- end custom head snippets -->

  </head>

  <body>
    <a id="skip-to-content" href="#content">Skip to the content.</a>

    <header class="page-header" role="banner">
      <h1 class="project-name">Welcome to Xie Lab!</h1>
      <h2 class="project-tagline">Jichun Xie, Associate Professor, Duke University</h2>
      
      <a href="./index.html" class="btn">Home</a>
      <a href="./research.html" class="btn">Research</a>
      <a href="./team.html" class="btn">Team</a>
      <a href="./publication.html" class="btn">Publication</a>
      <a href="./software.html" class="btn">Software</a>
      <a href="./courses.html" class="btn">Teaching</a>
    </header>

    <main id="content" class="main-content" role="main">
      <h1 id="software">Software</h1>

<ul>
  <li><strong>TEAM</strong>: A multiple testing algorithm on the aggregation tree for flow cytometry analysis.
    <ul>
      <li><a href="https://projecteuclid.org/journals/annals-of-applied-statistics/volume-17/issue-1/TEAM--A-multiple-testing-algorithm-on-the-aggregation-tree/10.1214/22-AOAS1645.short">paper</a> [AOAS, 2023]</li>
      <li><a href="https://cran.r-project.org/web/packages/TEAM/index.html">R package</a></li>
    </ul>
  </li>
  <li><strong>CDI</strong>: Clustering Deviation Index (CDI): A robust and accurate internal measure for evaluating scRNA-seq data clustering.
    <ul>
      <li><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02825-5">paper</a> [Genome Biology, 2022]</li>
      <li><a href="https://github.com/jichunxie/CDI">R package</a></li>
    </ul>
  </li>
  <li><strong>SQUAC</strong>: An efficient and robust large-scale multiple testing method to infer high-dimensional gene co-expression networks based on sample-quantile contingency tables.
    <ul>
      <li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6497089/">paper</a> [JSSSB, 2018]</li>
      <li><a href="https://github.com/jichunxie/squac">R codes</a></li>
    </ul>
  </li>
  <li><strong>bcSeq</strong>: This package implements a highly efficient data structure and algorithm for performing alignment of short reads from CRISPR or shRNA screens to reference barcode library. Sequencing error are considered and matching qualities are evaluated based on Phred scores. A Bayesâ€™ classifier is employed to predict the originating barcode of a read.
    <ul>
      <li><a href="https://academic.oup.com/bioinformatics/article/34/20/3581/5001385">paper</a> [Bioinformatics, 2018]</li>
      <li><a href="https://bioconductor.org/packages/release/bioc/html/bcSeq.html">R package</a></li>
    </ul>
  </li>
  <li><strong>capme</strong>: An efficient algorithm to estimate covariate-adjusted precision matrices.
    <ul>
      <li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5351557/">paper</a> [Biometrika, 2013]</li>
      <li><a href="https://github.com/cran/capme">R package</a></li>
    </ul>
  </li>
  <li><strong>qlspack</strong>: Fitting mixed effect model based on quasi-least-square.
    <ul>
      <li><a href="https://core.ac.uk/download/pdf/61321077.pdf">paper</a> [Statistics and Its Interface, 2010]</li>
      <li><a href="https://cran.r-project.org/src/contrib/Archive/qlspack/">R package</a></li>
    </ul>
  </li>
</ul>

    </main>

    <footer class="site-footer2">
        <div class=".site-footer2-contact-info">
          <strong>CONTACT</strong><br>
          Department of Biostatistics and Bioinformatics<br>
          Department of Mathematics<br>
          Duke University<br>
          Hock Plaza 10048<br>
          2424 Erwin Road<br>
          Durham, NC 27705<br>
          <span class="email-address">jichun DOT xie AT duke DOT edu</span>
        </div>
        <nav class="site-footer2-nav">
          <strong>NAVIGATION</strong><br><br>
          <ul>
            <li><a href="./index.html">Home</a></li><br>
            <li><a href="./research.html">Research</a></li><br>
            <li><a href="./team.html">Team</a></li><br>
            <li><a href="./publication.html">Publication</a></li><br>
            <li><a href="./software.html">Software</a></li><br>
            <li><a href="./courses.html">Teaching</a></li><br>
          </ul>
    </footer>     
  </body>
</html>